spec/services/recovery_code_generator_spec.rb

```diff
@@ -31,40 +31,42 @@ def stub_random_phrase
     end
 
     it 'generates a phrase of 4 words by default' do
-      user = create(:user)
-      generator = RecoveryCodeGenerator.new(user)
-
       expect(generator.create).to match(/\A\w\w\w\w \w\w\w\w \w\w\w\w \w\w\w\w\z/)
     end
 
     it 'allows length to be configured via ENV var' do
-      user = create(:user)
       allow(Figaro.env).to receive(:recovery_code_length).and_return('14')
-      generator = RecoveryCodeGenerator.new(user)
 
       expect(generator.create).to match(/\A(\w+\ ){13}\w+\z/)
```

Comment:
would it be possible to name this regex so mere humans can know know what it means at a glance? 🐰